dask_cuda/tests/test_dgx.py

```diff
@@ -102,9 +106,13 @@ def check_ucx_options():
 )
 def test_tcp_over_ucx(protocol):
     if protocol == "ucx":
-        pytest.importorskip("ucp")
+        ucp = pytest.importorskip("ucp")
     elif protocol == "ucxx":
-        pytest.importorskip("ucxx")
+        ucp = pytest.importorskip("ucxx")
+    if _is_ucx_116(ucp):
+        pytest.skip(
+            "Wireup may fail in UCX 1.16 in nodes with multiple NICs if TCP is used"
```

Comment:
Is there an Issue we can link to for the new skip logic?

Reply:
There wasn't one, I've filed https://github.com/rapidsai/ucx-py/issues/1037 to fill that gap and changed the skip message to point to that link via https://github.com/rapidsai/dask-cuda/pull/1331/commits/b0a86ab4e7d52b4d3092c8248a70ce23e2a28fb3 .